invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/index.js

```diff
@@ -45,6 +50,13 @@ class DashboardTabs extends Component {
   constructor(props) {
     super(props);
     const activeTabName = rootElement.dataset[_camelCase("active-tab-name")];
+    const communitiesEnabled =
+      rootElement.dataset[_camelCase("communities-enabled")] === "True";
```

Comment:
shouldn't it be already evaluated to `true` by JS json parse ? it is odd to compare with a string